packages/loader/container-loader/src/test/connectionStateHandler.spec.ts

```diff
@@ -933,220 +933,211 @@ describe("ConnectionStateHandler Tests", () => {
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.Connected,
-				"Client 3 should move to connected state",
-			);
-		},
-	);
-
-	it(
-		"Client 3 should wait for client 2(which got disconnected without sending any ops) to leave " +
-			"when client 2 already waited on client 1",
-		async () => {
-			connectionDetails.mode = "write";
-			connectionStateHandler.receivedConnectEvent(connectionDetails);
-			connectionStateHandler_receivedAddMemberEvent(pendingClientId);
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.Connected,
-				"Client 1 should be in connected state",
-			);
-
-			shouldClientJoinWrite = true;
-			// Disconnect the client
-			connectionStateHandler.receivedDisconnectEvent({ text: "Test" });
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.Disconnected,
-				"Client 1 should be in disconnected state",
-			);
-
-			// Make new client join but disconnect it from connected state
-			connectionStateHandler.receivedConnectEvent(connectionDetails2);
-			connectionStateHandler_receivedAddMemberEvent(pendingClientId2);
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.CatchingUp,
-				"Client 2 should still be in connecting state",
-			);
-			// Client 1 leaves.
-			connectionStateHandler_receivedRemoveMemberEvent(pendingClientId);
-
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.Connected,
-				"Client 2 should move to connected state",
-			);
-
-			// Client 2 leaves without sending any ops.
-			connectionStateHandler.receivedDisconnectEvent({ text: "Test" });
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.Disconnected,
-				"Client 2 should be in disconnected state",
-			);
-
-			// Make new client 3 join. Now it should not wait for previous client as client 2 already waited.
-			connectionStateHandler.receivedConnectEvent(connectionDetails3);
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.CatchingUp,
-				"Client 3 should still be in connecting state",
-			);
-			connectionStateHandler_receivedAddMemberEvent(pendingClientId3);
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.CatchingUp,
-				"Client 3 should still be in connecting state",
-			);
-
-			// Client 2 leaves.
-			connectionStateHandler_receivedRemoveMemberEvent("pendingClientId2");
-			assert.strictEqual(
-				connectionStateHandler.connectionState,
-				ConnectionState.Connected,
-				"Client 3 should move to connected state",
-			);
-			// Timeout should not raise any error as timer should be cleared
-			await tickClock(expectedTimeout);
-		},
-	);
+	it("Should wait for client 1 to leave before moving to connected state(Client 3) when client 2 " +
+		"got disconnected from connected state", async () => {
```

Comment:
In my opinion, this is much harder to read, since the body of the function has the same indent as the line before it and the function declaration is at the end of a line after unrelated content. This really blurs the function body with the test arguments in a hard to visually parse way.

Reply:
Seems like the same formatting bug we hit with the tree test, just worse since the first thing in the lambda isn't a loop.

Reply:
I filed https://github.com/biomejs/biome/issues/4202 along with a playground link that demonstrates the issue. It looks like prettier 3.3.3 formats as we'd prefer (according to the playground).